mm2src/coins/eth.rs

```diff
@@ -182,39 +199,27 @@ lazy_static! {
 
 pub type Web3RpcFut<T> = Box<dyn Future<Item = T, Error = MmError<Web3RpcError>> + Send>;
 pub type Web3RpcResult<T> = Result<T, MmError<Web3RpcError>>;
-pub type GasStationResult = Result<GasStationData, MmError<GasStationReqErr>>;
 type EthPrivKeyPolicy = PrivKeyPolicy<KeyPair>;
-type GasDetails = (U256, U256);
 
-#[derive(Debug, Display)]
-pub enum GasStationReqErr {
-    #[display(fmt = "Transport '{}' error: {}", uri, error)]
-    Transport {
-        uri: String,
-        error: String,
-    },
-    #[display(fmt = "Invalid response: {}", _0)]
-    InvalidResponse(String),
-    Internal(String),
+#[derive(Clone, Debug)]
+pub(crate) struct LegacyGasPrice {
+    pub gas_price: U256,
 }
 
-impl From<serde_json::Error> for GasStationReqErr {
-    fn from(e: serde_json::Error) -> Self { GasStationReqErr::InvalidResponse(e.to_string()) }
+#[derive(Clone, Debug)]
+pub(crate) struct Eip1559FeePerGas {
+    pub max_priority_fee_per_gas: U256,
+    pub max_fee_per_gas: U256,
```

Comment:
same as above, if you use `pub(crate)` visibility level for struct, fields should have the same lvl or more restrictive.

Reply:
done in [0ec27e9](https://github.com/KomodoPlatform/komodo-defi-framework/pull/2051/commits/0ec27e914bd8d3eb0dd40d1ff9500dad459ff742)